app/scripts/lib/ens-ipfs/resolver.js

```diff
@@ -61,7 +61,7 @@ function getRegistryForChainId (chainId) {
   switch (chainId) {
     // mainnet
     case 1:
-      return '0x314159265dd8dbb310642f98f50c066173c1259b'
+      return '0x00000000000C2E074eC69A0dFb2997BA6C7d2e1e'
     // ropsten
     case 3:
       return '0x112234455c3a32fd11230c42e7bccd4a84e02010'
```

Comment:
What about the other networks?

>All ENS deployments across mainnet and all testnets now use the same addresses.

Reply:
Huh, so that _was_ intentional. I missed that bit but did notice that the @ensbot PR changed them all. I'll update this.